lava/utils.py

```diff
@@ -114,27 +113,30 @@ async def get_recommended_tracks(player: "LavaPlayer", track: AudioTrack, max_re
     :param track: The seed tracks to get recommended tracks from.
     :param max_results: The max amount of tracks to get.
     """
-    try:
-        results_from_youtube = await youtube_related.async_fetch(track.uri)
-    except ValueError:  # The track is not a YouTube track
-        search_results = youtube_search.YoutubeSearch(f"{track.title} by {track.author}", 1).to_dict()
+    results_from_yt: LoadResult = None
 
-        results_from_youtube = await youtube_related.async_fetch(
-            f"https://youtube.com/watch?v={search_results[0]['id']}"
-        )
+    if track.source_name != "youtube":
+        videosSearch = VideosSearch(f"{track.title} by {track.author}", limit=1)
```

Comment:
As of PEP-8, it's recommended to use snake_case when introducing variables. Considering renaming it here?